### ISSUE TITLE
Release Diffusion generated synthetic Dataset on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [shyammarjit/LR0.FM#1]
Hello @ppriyank 🤗 

Niels here from the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to hf.co/papers to improve its discoverability. If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss about your paper and lets them find artifacts about it (your dataset for instance),
you can also claim the paper as yours which will show up on your public profile at HF.

Would you like to host the datasets you've released on https://huggingface.co/datasets?
I see you're using Github for it. Hosting on Hugging Face will give you more visibility/enable better discoverability, and will also allow people to do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```

If you're down, leaving a guide here: https://huggingface.co/docs/datasets/loading.
We also support Webdataset, useful for image/video datasets: https://huggingface.co/docs/datasets/en/loading#webdataset.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

After uploaded, we can also link the datasets to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

Let me know if you're interested/need any guidance.

Kind regards,

Niels

**Comment by ppriyank** [comment]
Sure, I will upload the pre-trained weights and models on Hugging Face.

**Comment by ppriyank** [comment]
@NielsRogge  Can you help us set up this benchmarking on Hugging Face? We have results/ranking for 66 models with our evaluation metric on the standard dataset. 
A new zero-shot classifier can automatically be ranked in the future if the benchmarking is live.

**Comment by NielsRogge** [comment]
Yes, definitely! Would you like to share this as a dataset on [hf.co/datasets](hf.co/datasets)? Or do you want to create a demo with Gradio/Streamlit/custom Docker on [hf.co/spaces](hf.co/spaces)?

**Comment by ppriyank** [comment]
Hello @NielsRogge 

We have been trying to figure out what to upload to Hugging Face. 
We can upload ranking of models, will that be acceptable? 


Synthetic Dataset: It's 872 GB, which can easily be generated using the script provided (within a few hours) using the captions provided. 

Model Weights: We can train model, but three models (EVA, OpenCLIP, and MetaCLIP). But purpose will that server? Zero-shot classification task may be too much of a toy problem of the Real-world.

**Comment by NielsRogge** [comment]
Hi @ppriyank, it's of course not an obligation to upload artifacts if you don't have any :)

Perhaps it might be useful to upload the captions to the hub: https://github.com/shyammarjit/LR0.FM/tree/main/MetaData/Captions, they can be made compatible with the Datasets library as seen here: https://huggingface.co/docs/datasets/loading#local-and-remote-files

**Comment by NielsRogge** [comment]
Also I've indexed your paper here: https://huggingface.co/papers/2502.03950

Feel free to claim it with your HF account and add the Github and/or project page URLs

**Comment by ppriyank** [comment]
Thank you so much!! 

I think once my request is approved to claim ownership of the paper, I can add links to the project page and GitHub.
We will upload the caption.
